<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project_before</class>
 <widget class="QWidget" name="project_before">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#project_before
{
background-color:black;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
color:white;
}</string>
   </property>
   <property name="text">
    <string>What do you want to do related to the project?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_task_person_pbn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#change_task_person_pbn{
background-color:white;
border-radius:10px;
font-size:20px;
}

#change_task_person_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Changing the task of the member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_task_person_pbn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>291</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_task_person_pbn{
background-color:white;
border-radius:10px;
font-size:20px;
}

#view_task_person_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing task of member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_admin_pbn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#add_admin_pbn{
background-color:white;
border-radius:10px;
font-size:25px;

}

#add_admin_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Adding admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_admin_pbn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#delete_admin_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#delete_admin_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Deleting admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_admin_pbn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_admin_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#view_admin_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing admins</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_member_pbn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#add_member_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#add_member_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Adding member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_member_pbn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#delete_member_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#delete_member_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Deleting member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_member_pbn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_member_pbn{
background-color:white;
border-radius:10px;
font-size:25px;
}

#view_member_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_project_task_pbn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>550</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#set_project_task_pbn{
background-color:white;
border-radius:10px;
font-size:24px;
}

#set_project_task_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Setting task for project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_project_task_pbn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>420</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_project_task_pbn{
background-color:white;
border-radius:10px;
font-size:24px;
}

#view_project_task_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing task of project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_project_status_pbn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>310</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#change_project_status_pbn{
background-color:white;
border-radius:10px;
font-size:20px;
}

#change_project_status_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Changing status of project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_project_status_pbn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>210</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#view_project_status_pbn{
background-color:white;
border-radius:10px;
font-size:22px;
}

#view_project_status_pbn:hover
{
background-color:yellow;
order:1px solid yellow; 
}</string>
   </property>
   <property name="text">
    <string>Viewing status of project</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_add_adm">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_del_adm">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_add_mem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_del_mem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_set_taskpro">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>520</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter task</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_view_taskpro">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>390</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_chang_status">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>280</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_view_status">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>180</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_chan_taskmem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter task</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_view_taskmem">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>50</y>
     <width>291</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ted_view_adm">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ted_view_mem">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbldel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lbldel{
background-color:white;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbladd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>530</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lbladd{
background-color:white;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_set_tasktimepro">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>490</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_chan_tasktimemem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_namechange">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_nameforview">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>291</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
